tensorflow_estimator/tools/pip_package/setup.py

```diff
@@ -30,7 +30,7 @@
 # This version string is semver compatible, but incompatible with pip.
 # For pip, we will remove all '-' characters from this string, and use the
 # result for pip.
-_VERSION = '2.11.0'
+_VERSION = '2.12.0'
```

Comment:
```suggestion
_VERSION = '2.11.0-rc0'
```

(this is for RC0 release)